omisego-sdk/src/test/kotlin/co/omisego/omisego/OMGAPIClientTest.kt

```diff
@@ -145,7 +147,7 @@ class OMGAPIClientTest {
     @Test
     fun `OMGAPIClient should be executed when API return success true correctly`() {
         val expected = gson.fromJson<OMGResponse<User>>(userFile.readText(),
-                object : TypeToken<OMGResponse<User>>() {}.type)
+            object : TypeToken<OMGResponse<User>>() {}.type)
```

Comment:
What do you think of
```kotlin
val expected = gson.fromJson<OMGResponse<User>>(
    userFile.readText(),
    object : TypeToken<OMGResponse<User>>() {}.type
)
```

This way, arugments would be aligned, easier to read 🤔

Reply:
Fixed